Makefile

```diff
@@ -34,23 +34,32 @@ create_db: start
 .PHONY: db_diagram
 db_diagram:
 	# Create new dot file showing current version of schema
-	eralchemy -i postgresql://openoversight:terriblepassword@postgres/openoversight-dev -o database/schema.new.dot
+	eralchemy2 -i postgresql://openoversight:terriblepassword@postgres/openoversight-dev -o database/schema.new.md
```

Comment:
The markdown files were more readable and provided a more consistent result on my local.